graph/client/src/app/feature-tasks/task-list.tsx

```diff
@@ -44,7 +44,7 @@ function ProjectListItem({
 
       {project.error ? (
         <Tooltip
-          content={<TaskGraphErrorTooltip error={project.error} />}
+          content={(<TaskGraphErrorTooltip error={project.error} />) as any}
```

Comment:
Maybe `JSX.Element` is the correct type here? I've always used `ReactNode` since it's more flexible, but perhaps the updated type is doing something unexpected.

Reply:
@philipjfulcher wdyt?